Formula/opencv.rb

```diff
@@ -98,6 +98,8 @@ def install
       system "cmake", "..", "-DBUILD_SHARED_LIBS=OFF", *args
       system "make"
       lib.install Dir["lib/*.a"]
+      lib.install Dir["3rdparty/lib/*.a"]
```

Comment:
Does `lib.install Dir["3rdparty/lib/**/*.a"]` grab them all?

Reply:
or even `lib.install Dir["3rdparty/**/*.a"]`

Reply:
Let me test that.